ballerina/modules/oas/client.bal

```diff
@@ -0,0 +1,2701 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is auto-generated by the Ballerina OpenAPI tool.
```

Comment:
better to have WSO2 license header for the generated files as well. You can use `--license` options in the openAPI tool when generating the client code.

Reply:
Sure.Addressed in [5d64220](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/5d64220725ae541b41890724532148c1a2154da8).